### PR TITLE
[ZH-TW] Update `Main page`

### PR DESCRIPTION
**Pull request by 5idereal**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Main_page/zh-tw.md [hunk 1]
```diff
@@ -101,9 +99,11 @@ outdated_translation: true
 
 社交互動是推動世界各地的 osu! 玩家的事情之一。探索各類活動及競賽，與他人交流，或是協助籌辦活動！
 
-[錦標賽](/wiki/Tournaments) • [圖譜聚光燈](/wiki/Beatmap_Spotlights) • [製譜競賽](/wiki/Contests) • [社群師徒計畫](/wiki/Community/Community_Mentorship_Program)
+平台：[Forum](/wiki/Community/Forum) • [IRC](/wiki/Community/Internet_Relay_Chat) • [Discord 伺服器](/wiki/Community/Discord_servers)
```
Comment (YumeMuzi) [inline wiki/Main_page/zh-tw.md:102]:
```suggestion
平台：[論壇](/wiki/Community/Forum) • [IRC](/wiki/Community/Internet_Relay_Chat) • [Discord 伺服器](/wiki/Community/Discord_servers)
```

**Review by YumeMuzi — CHANGES_REQUESTED** [review]
Forum 还是可以翻译一下的

File: wiki/Main_page/zh-tw.md (hunk 1 above)
Comment (5idereal) [reply wiki/Main_page/zh-tw.md:102]: 沒檢查到🙇 
感謝 review!

File: wiki/Main_page/zh-tw.md [hunk 2]
```diff
@@ -64,7 +62,7 @@ outdated_translation: true
 
 審核是指圖譜取得排行榜的過程，可使圖譜更受歡迎並獲得認可。了解所需條件以及如何提高您通過的機率，還有品質控管及審核流程的程序。
 
-[提交](/wiki/Beatmapping/Beatmap_submission) • [Modding](/wiki/Modding) • [審核流程](/wiki/Beatmap_ranking_procedure) • [製譜者公會 (Mappers' Guild)](/wiki/Community/Mappers_Guild) • [Project Loved](/wiki/Community/Project_Loved)
+[提交](/wiki/Beatmapping/Beatmap_submission) • [Mod 圖](/wiki/Modding) • [審核流程](/wiki/Beatmap_ranking_procedure) • [製圖者公會 (Mappers' Guild)](/wiki/Community/Mappers_Guild) • [Project Loved](/wiki/Community/Project_Loved)
```
Comment (rucker122) [inline wiki/Main_page/zh-tw.md:65]: Modding 可翻成摸圖

File: wiki/Main_page/zh-tw.md [hunk 3]
```diff
@@ -101,9 +99,11 @@ outdated_translation: true
 
 社交互動是推動世界各地的 osu! 玩家的事情之一。探索各類活動及競賽，與他人交流，或是協助籌辦活動！
 
-[錦標賽](/wiki/Tournaments) • [圖譜聚光燈](/wiki/Beatmap_Spotlights) • [製譜競賽](/wiki/Contests) • [社群師徒計畫](/wiki/Community/Community_Mentorship_Program)
+平台：[論壇](/wiki/Community/Forum) • [IRC](/wiki/Community/Internet_Relay_Chat) • [Discord 伺服器](/wiki/Community/Discord_servers)
 
-[專案](/wiki/Community/Projects): [精選藝術家](/wiki/People/Featured_Artists) • [osu! 社群會議](/wiki/Community/osu!_community_meetings) • [osu!academy](/wiki/Community/Video_series/osu!academy) • [osu!mapping](/wiki/Community/Video_series/osu!mapping) • [osu!talk](/wiki/Community/Video_series/osu!talk)
+[錦標賽](/wiki/Tournaments) • [圖譜聚光燈](/wiki/Beatmap_Spotlights) • [製圖競賽](/wiki/Contests) • [社群師徒計畫](/wiki/Community/Community_Mentorship_Program)
```
Comment (rucker122) [inline wiki/Main_page/zh-tw.md:104]: Tournament 翻成賽事或比賽比較直接

File: wiki/Main_page/zh-tw.md [hunk 4]
```diff
@@ -133,8 +133,8 @@ outdated_translation: true
 # Wiki 及描述頁
 
 如同這裡幾乎所有的一切，osu! wiki 也是由志願者撰寫並維護的。歡迎在 [osu-wiki GitHub 儲存庫](https://github.com/ppy/osu-wiki)提出建議、回報問題、甚至自行撰寫文章。
-    
-[osu! 歷史](/wiki/History_of_osu!) • [osu! wiki 歷史](/wiki/History_of_osu!/osu!_wiki) • [製譜及 Modding 時間線](/wiki/History_of_osu!/Mapping_and_modding_timeline) • [愚人節](/wiki/History_of_osu!/April_Fools)
+
+[osu! 歷史](/wiki/History_of_osu!) • [線上排行榜歷史](/wiki/History_of_osu!/Online_rankings) • [osu! wiki 歷史](/wiki/History_of_osu!/osu!_wiki) • [製圖及 Mod 圖時間線](/wiki/History_of_osu!/Mapping_and_modding_timeline) • [愚人節](/wiki/History_of_osu!/April_Fools)
```
Comment (rucker122) [inline wiki/Main_page/zh-tw.md:137]: 玩家用語可能 排名 會比 排行榜 常見, 但不太一定
這裡用排名歷史看起來比較順

**Comment by 5idereal** [comment]
@rucker122 已按照您的建議做修改了，感謝 review!

**Review by rucker122 — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**